### PR TITLE
D ren seq modifications

### PR DESCRIPTION
**Pull request by TMAdams**
In progress PR for modifications to dRenSeq workflow

### CODE REVIEW

**Comment by TMAdams** [comment]
This has run successfully, will update main.nf in a subsequent PR as I don't want to do more merge conflicts

File: modules/drenseq.nf [hunk 1]
```diff
@@ -104,162 +86,307 @@ process BowtieAlign {
...
-    library(tidyverse)
+    bedtools coverage -d \
+        -a $bait_regions_bed \
+        -b $bam \
+        > coverage.txt
+    """
+}
 
-    EXTENSION <- ".coverage.txt"
+process PerGeneCoverage {
```
Comment (SwiftSeal) [inline modules/drenseq.nf:301]: diabolical bash

Comment (SwiftSeal) [reply modules/drenseq.nf:301]: but fine lol

Comment (SwiftSeal) [reply modules/drenseq.nf:301]: actually what was wrong with the R script :'(

**Review by SwiftSeal — APPROVED** [review]

File: modules/drenseq.nf (hunk 1 above)
Comment (TMAdams) [reply modules/drenseq.nf:301]: I didn't see how it was going to produce the same output with the same checks as the old workflow, so just went back to what we had in HISS

**Comment by TMAdams** [comment]
Did a resource efficiency pass. Running a test elsewhere with dynamic time allocation in addition to dynamic memory allocation. When this works I'll add a PR doing it for all workflows

**Review by SwiftSeal — APPROVED** [review]

**Pull request merged**